lua/lazyvim/plugins/extras/lang/laravel.lua

```diff
@@ -0,0 +1,199 @@
+      },
+      {
+        "<leader>la",
+        function()
+          Laravel.pickers.artisan()
+        end,
+        desc = "Laravel: Open Artisan Picker",
+      },
+      {
+        "<leader>lt",
+        function()
+          Laravel.commands.run("actions")
+        end,
+        desc = "Laravel: Open Actions Picker",
+      },
+      {
+        "<leader>lr",
+        function()
+          Laravel.pickers.routes()
+        end,
+        desc = "Laravel: Open Routes Picker",
+      },
+      {
+        "<leader>lh",
+        function()
+          Laravel.run("artisan docs")
+        end,
+        desc = "Laravel: Open Documentation",
+      },
+      {
+        "<leader>lm",
+        function()
+          Laravel.pickers.make()
+        end,
+        desc = "Laravel: Open Make Picker",
+      },
+      {
+        "<leader>lc",
+        function()
+          Laravel.pickers.commands()
+        end,
+        desc = "Laravel: Open Commands Picker",
+      },
+      {
+        "<leader>lo",
+        function()
+          Laravel.pickers.resources()
+        end,
+        desc = "Laravel: Open Resources Picker",
+      },
+      {
+        "<leader>lp",
+        function()
+          Laravel.commands.run("command_center")
+        end,
+        desc = "Laravel: Open Command Center",
+      },
+      {
+        "gf",
+        function()
+          local ok, res = pcall(function()
+            if Laravel.app("gf").cursorOnResource() then
+              return "<cmd>lua Laravel.commands.run('gf')<cr>"
+            end
+          end)
+          if not ok or not res then
+            return "gf"
+          end
+          return res
+        end,
+        expr = true,
+        noremap = true,
+      },
+    },
+    event = { "VeryLazy" },
+    opts = {
+      features = {
+        pickers = {
+          provider = "snacks", -- "snacks | telescope | fzf-lua | ui-select"
+        },
+      },
+    },
+  },
+
+  {
+    "mason-org/mason.nvim",
+    opts = {
+      ensure_installed = {
+        "blade-formatter",
+        "pint",
+        "rustywind",
+      },
+    },
+  },
+
+  {
+    "mfussenegger/nvim-dap",
+    optional = true,
+    opts = function()
+      local dap = require("dap")
+      dap.adapters.php = {
+        type = "executable",
+        command = "php-debug-adapter",
+        args = {},
+      }
+      dap.configurations.php = {
+        {
+          type = "php",
+          request = "launch",
+          name = "Listen for Xdebug",
+          port = 9003,
+          pathMappings = {
+            ["/var/www/html"] = "${workspaceFolder}",
+          },
+        },
+      }
+    end,
+  },
+
+  {
+    "nvim-neotest/neotest",
+    optional = true,
+    dependencies = {
+      "V13Axel/neotest-pest",
+      "olimorris/neotest-phpunit",
+    },
+    opts = {
+      adapters = {
+        "neotest-pest",
+        ["neotest-phpunit"] = {
+          root_ignore_files = { "tests/Pest.php" },
+        },
+      },
+    },
+  },
```

Comment:
These are duplicated with the PHP Extra. I assume one will use Laravel with PHP Extra also enabled.

Reply:
Yes, you can do it like in the svelte Extra. See [here](https://github.com/LazyVim/LazyVim/blob/c64a61734fc9d45470a72603395c02137802bc6f/lua/lazyvim/plugins/extras/lang/svelte.lua#L14), where it imports the Typescript Extra when you enable svelte Extra.